src/components/AnalysisPage/AnalysisPage.jsx

```diff
@@ -10,13 +10,22 @@ const AnalysisPage = () => {
   const [fileFor3dModel, setFileFor3dModel] = useState(null);
   const [partId, setPartId] = useState(null);
   const [autoPopAnalysis, setAutoPopAnalysis] = useState(null);
+  const [analysisComplete, setAnalysisComplete] = useState(false);
+  const [showCheckmark, setShowCheckmark] = useState(false);
```

Comment:
I must at one point have been passing these two pieces of state down to and using them in ViewPartsButton, 

But that changed along the way, and I never moved where they were defined because you're right. There's no need for them to be in the analysis page the way they're being set up and used.